Version3

ruslankorotkov opened the pull request.

ruslankorotkov committed:
программа, которая проверяет, является ли строка палиндромом. применение метода.

ruslankorotkov committed:
программа, которая проверяет, является ли строка палиндромом. применение метода.применение классов.

ruslankorotkov committed:
КУРСОВАЯ РАБОТА.

ruslankorotkov committed:
программа, которая проверяет, является ли строка палиндромом. применение метода.применение классов.

ruslankorotkov committed:
КУРСОВАЯ РАБОТА.

ruslankorotkov committed:
КУРСОВАЯ РАБОТА.

ruslankorotkov committed:
КУРСОВАЯ РАБОТА.

dmitry-bizin commented on src/pro/sky/java/course1/coursework3/Main.java:
```diff
@@ -0,0 +1,52 @@
+package pro.sky.java.course1.coursework3;
+
+public class Main {
+    private static final Employee[] staff = {
```
поля static final принято называть в верхнем регистре

dmitry-bizin replied:
т.е. STAFF

dmitry-bizin commented on src/pro/sky/java/course1/coursework3/Main.java:
```diff
@@ -0,0 +1,52 @@
+        for (Employee element : staff) {
+            total += element.getSalary();
+        }
+        return total / staff.length;
```
деление нацело, т.е. теряется остаток. А среднее - это вещественное число, а не целое

dmitry-bizin commented on src/pro/sky/java/course1/coursework3/Main.java:
```diff
@@ -0,0 +1,52 @@
+        return total / staff.length;
+    }
+
+
```
а где методы поиска сотрудника с мин и макс ЗП? всего должно быть 6 методов в базовой части

dmitry-bizin commented on src/pro/sky/java/course1/coursework3/Main.java:
```diff
@@ -0,0 +1,52 @@
+    public static int searchAverageSalary() {
+        int total = 0;
+        for (Employee element : staff) {
+            total += element.getSalary();
```
зачем дублировать код, если у тебя уже есть метод, который считает сумму? надо его вызвать и поделить **_правильно_** на длину массива

ruslankorotkov committed:
КУРСОВАЯ РАБОТА. исправил.
методы поиска минимальной и максимальной зарплаты не получаются.
метод где выводится  result нам вроде как не объясняли.

ruslankorotkov committed:
КУРСОВАЯ РАБОТА. (исправил что смог )
методы поиска минимальной и максимальной зарплаты не получаются. делал по аналогии как делали д.з. с массивами ( принимая во внимание видеоразбор курсовой ).
В курсовой работе
метод где выводится result нам вроде как не объясняли( я result увидил в видеоразборе курсовой).
Подскажите пожалуйста как сделать правильно.
[ ребята у меня к вам предложение: вы меня нормально учите, а я вам раскажу про стартап { Знакомства с гарантией совместимости 100% для гетеросексуалов(в основе лежит физиология человека\запах\ }который вы можете сделать]
